planet/src/main/webapp/places.js

```diff
@@ -102,39 +108,46 @@ function initMap() {
     });
 }
 
+function handleLocationError(browserHasGeolocation, infoWindow, pos) {
```

Comment:
It looks like we only support the case where the user has location enabled. Is there a way we can gracefully degrade if they have it disabled? Like allowing them to search?

Reply:
The user is still able to search even if they don't have location allowed

Reply:
Right now, there is an error message that pops up as an info window marker on the map - when that feature is improved I will make changes accordingly but we still aren't sure what to do with the user location